AdventureWorksETL/AdventureWorksETL.dtproj

```diff
@@ -26,12 +26,13 @@
           <SSIS:Property SSIS:Name="CreatorComputerName">WIN-IMLUOESP5LO</SSIS:Property>
           <SSIS:Property SSIS:Name="Description">
           </SSIS:Property>
-          <SSIS:Property SSIS:Name="PasswordVerifier" SSIS:Sensitive="1">AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAA4tHesL6j2kuHhX0yOKgDSAAAAAACAAAAAAAQZgAAAAEAACAAAAAh3JrW6K2oUAnBRwMqeZq8J7Fcf4JRS3U5czts1eDWuwAAAAAOgAAAAAIAACAAAADSp41oxIw4dh6otzVDaKs26BX7IA/hXNeXuADQAYyWBpAAAABk4XETFGen55ycGvApER7pTtADivnM/fWWMXcurt1Ut8nTwdhap+FHrHxpM1kaHxAbfE/lXNYQeNXx7WgL/djpBetTJ5LNKOfaJvczpQa0dMTSdPNyQgLyjt1f3FAMHx7AFJoEBQoAqE3+eOLfc0AgfXMZcxcjkYXhDtIYrsIFxQj449ofA9kT+vnEie0MmNBAAAAAKYYeyQPeHrJGncR6d63GR0zvV43BAqN+2GnfL+RaSAj0qcMtp6Qhl1jBfJ87ucxjAqI6yuM7kNkPSO69Vvza2Q==</SSIS:Property>
+          <SSIS:Property SSIS:Name="PasswordVerifier" SSIS:Sensitive="1">AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAA4tHesL6j2kuHhX0yOKgDSAAAAAACAAAAAAAQZgAAAAEAACAAAABpRCanI87wVte4zsHdMRzobW/mZ7u1HE/CfDfegJ221QAAAAAOgAAAAAIAACAAAAB3uV9AmG+sDip/qqZHD029wWvym04lKbfqEFIgaza9sZAAAACa+rQTPipKdfmYCen/XJjeygp34mEY6b+GonAhU4FLocLmgQv3456p52WGLGwom48noEAw73Mj5zFiRqY12cOZ5nq5yFy3gbVFEcUqZ6DIeW8kg33lHW3L0ngI2WIzxtK62WjhGqkRLlKpS458KowYSb3kx+8hjbYbE5DvBkF8hMKaDeQEEbeEk46H72WezCxAAAAAg9M5mU+SJPaX0rbtQA99nATNsAChXnRaUp5NEL70+HXHBNisCxUKxsLBqlR48zWueuEQcztTT1LOzeDafvTHwQ==</SSIS:Property>
```

Comment:
Een probleem waar we wel met git tegenaan gaan lopen, is dat dit encrypted data is die per gebruiker anders is. Ik kan dit dus niet lokaal clonen en dit decrypten. Denk niet dat dat een issue is aangezien ik zelf gewoon het wachtwoord kan invullen, maar even ter informatie

Reply:
Wordt opgelost met een gedeelde sleutel zoals Teun al aangaf in Teams